content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -8,10 +8,12 @@ card:
 ---
 
 <!-- overview -->
+Many applications rely on configuration which is used during either application initialization or runtime.
```

Comment:
How about this? 

```suggestion
Many applications rely on configuration data that can be easily customized and managed depending on the environment. 
ConfigMaps are a great way to create tailored running services from generic container images by injecting configuration data into a container when it starts up.
```

Reply:
If this suggestion is accepted, we can delete the next two lines sine they'd be repetitive.

Reply:
I think it's helpful to emphasize that Pods can consume ConfigMaps both during startup and during their running phase. A reader who misunderstood might think that ConfigMaps are only relevant for the running phase, and (start to) design their own mechanism for applying configuration during app init.